demos/SnakeGame/Services/IDrawable.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using SnakeGame.Models;
+
+namespace SnakeGame.Services
+{
+    public interface IDrawable
+    {
+        void Draw();
+
+        static void Draw(Position p, ConsoleColor color)
```

Comment:
You are putting a method in an interface. So it's time for you to use `abstract class`